### PR TITLE
[Merged by Bors] - feat(RingTheory/PiTensorProduct): extensionality and isomorphisms

### PR DESCRIPTION
**Pull request by eric-wieser**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/PiTensorProduct.lean [hunk 1]
```diff
@@ -217,6 +250,53 @@ instance instCommSemiring : CommSemiring (⨂[R] i, A i) where
...
+
+variable {R ι}
+
+@[simp]
+theorem constantBaseRingEquiv_tprod (x : ι → R) :
+    constantBaseRingEquiv ι R (tprod R x) = ∏ i, x i := by
+  simp only [constantBaseRingEquiv, LinearEquiv.ofLinear_apply, lift.tprod,
+    MultilinearMap.mkPiAlgebra_apply]
+  simp [AlgEquiv.ofAlgHom]
+
```
[the remark this answers is not in the document]
Comment (eric-wieser) [reply Mathlib/RingTheory/PiTensorProduct.lean]: I added a missing `@[simps]` and remove one of those lemmas.

[the remark this answers is not in the document]
Comment (jjaassoonn) [reply Mathlib/RingTheory/PiTensorProduct.lean]: thanks!

**Comment by smorel394** [comment]
This looks good to me.

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**